__tests__/app.test.js

```diff
@@ -404,4 +404,44 @@ describe("PATCH /api/articles/:article_id", () => {
         expect(body.msg).toBe("Bad request");
       });
   });
+  test("should respond with 400 status if inc_votes is missing or not a number", () => {
```

Comment:
This would be a different error 
Status 200, if inc_votes is missing, should not update article and return original single article object (this may be included as a 400 but that could make for a brittle test suite)
Status 400, incorrect body, e.g. inc_votes property is not a number